docs/api/flink/Function.md

```diff
@@ -1244,6 +1244,19 @@ Result:
 +------------------+------------------------+
 ```
 
+## ST_MinimumBoundingCircle
+
+Introduction: Returns the smallest circle polygon that contains a geometry. The optional quadrantSegments parameter determines how many segments to use per quadrant and the default number of segments is 48. 
+
+Format: `ST_MinimumBoundingCircle(geom: geometry, [Optional] quadrantSegments:int)`
+
+Since: `v1.0.1`
```

Comment:
Why is this version reverted back to v1.0.1? All these functions should be v1.5.0 @yyy1000

Reply:
Ah, sorry for missing that. Updated.